index.html

```diff
@@ -6664,11 +6672,11 @@ <h5>Presentational Role Inheritance</h5>
 						<td><rref>list</rref></td>
 					</tr>
 					<tr>
-						<th class="role-scope-head" scope="row">Required Context Role:</th>
+						<th class="role-scope-head" scope="row">Required Parent Role:</th>
 						<td class="role-scope"> </td>
 					</tr>
 					<tr>
-						<th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+						<th class="role-mustcontain-head" scope="row">Allowed Child Roles:</th>
 						<td class="role-mustcontain"><rref>radio</rref></td>
```

Comment:
This one seems wrong now. Radiogroups do not require radio nodes as children, any descendant is fine. This is the only one of the required owned elements where I think it made sense to 1. require at least one, and 2. allow descendants.

If you wanted to keep it, you could add a MUST or SHOULD to L6642. In either case, this should come out.

```suggestion
						<td class="role-mustcontain"> </td>
```

Reply:
Good catch, I'll remove the allowed child role from `radiogroup`.